<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>GameForm</class>
 <widget class="QWidget" name="GameForm">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>540</width>
    <height>290</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Bomberbot</string>
  </property>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>0</y>
     <width>521</width>
     <height>281</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="main_layout">
    <property name="margin">
     <number>5</number>
    </property>
    <item>
     <layout class="QVBoxLayout" name="field_layout">
      <item>
       <widget class="QLabel" name="log_label">
        <property name="text">
         <string>Log:</string>
        </property>
        <property name="buddy">
         <cstring>textBrowser</cstring>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QTextBrowser" name="textBrowser">
        <property name="maximumSize">
         <size>
          <width>16777215</width>
          <height>80</height>
         </size>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QVBoxLayout" name="info_layout">
      <item>
       <widget class="QLabel" name="players_label">
        <property name="text">
         <string>Players:</string>
        </property>
        <property name="buddy">
         <cstring>players_table</cstring>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QTableView" name="players_table"/>
      </item>
      <item>
       <widget class="QLabel" name="bonuses_label">
        <property name="text">
         <string>Bonuses:</string>
        </property>
        <property name="buddy">
         <cstring>bonuses_table</cstring>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QTableView" name="bonuses_table"/>
      </item>
      <item>
       <widget class="QLabel" name="bombs_label">
        <property name="text">
         <string>Bombs</string>
        </property>
        <property name="buddy">
         <cstring>bombs_table</cstring>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QListView" name="bombs_table"/>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
